src/ILCompiler.ReadyToRun/src/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -6,6 +6,7 @@
 using System.Diagnostics;
 
 using Internal.IL;
+using Internal.JitInterface;
```

Comment:
Why do we need this?

Reply:
Removed in the trylek/tree/RemainingR2RChanges branch.